### PR TITLE
clarify how a term `payload` is used

### PR DESCRIPTION
**Pull request by Sakurann**
As pointed out in Issuer #26, a term payload has been used to characterize an object with a set of claims that the Issuer is trying to disclose using SD-JWT. Also clarifies when it is an example of an entire SD-JWT payload or just an `_sd` object.

Note: I would suggest using only Example 1 of 'flat' SD-JWT in the main text and move Example 2 to the Annex.

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -698,7 +705,7 @@ TBD
 
 In this example, a complex object such as those used for ekyc (todo reference) is used.
 
-These claims are the payload of the SD-JWT:
+In this example, the Issuer is using a following object as a set of claims to issue to the Holder:
```
Comment (danielfett) [inline main.md:708]:
```suggestion
In this example, the Issuer is using the following object as a set of claims to issue to the Holder:
```

File: main.md [hunk 2]
```diff
@@ -244,15 +232,23 @@ Note: need to define how holder public key is included, right now examples are u
...
+hashed claim values without any deeper structure. The `_sd` object can also be a
+'structured' object, where some claims and their respective hashes are contained
+in places deeper in the structure. it is at the issuer's discretion whether to use
+a 'flat' or 'structured' `_sd` SD-JWT object, and how to structure it such that
+it is suitable for the use case.
 
-In this example, these claims are the payload of the SD-JWT:
+Examples 1 is a non-normative example of an SD-JWT using a 'flat' `_sd` object
+and example 2 is a non-normative example of an SD-JWT using a 'structured' `_sd` object.
+The difference between the examples is how an `address` claim is disclosed.
```
Comment (danielfett) [inline main.md:249]:
```suggestion
The difference between the examples is how the `address` claim is disclosed.
```

**Review by danielfett — CHANGES_REQUESTED** [review]

**Comment by selfissued** [comment]
I agree with @danielfett 's suggested changes.

**Review by selfissued — APPROVED** [review]
I'm good to go with this once @danielfett 's suggestions have been incorporated.

**Review by selfissued — APPROVED** [review]
Please see the content of my previous approval.  I clicked the wrong part of the page and cleared my "reviewed" status.

**Review by bc-pi — APPROVED** [review]

**Pull request merged**